### ISSUE TITLE
Add support for shuffling your whole library

### ISSUE THREAD

**Issue by ScaleRunner** [leinelissen/jellyfin-audio-player#16]
First of all, thanks for making this app, it works great in what it already does! 

The main way I listen to my music, is to just shuffle my whole library. I would like to see this feature in the app.

Keep up the good work 👍🏼

**Comment by leinelissen** [comment]
Right! I am looking into shuffling at the moment, but the implementation appears kinda iffy. Doing it for the whole library seems more doable, but it requires implementing track view for the whole library. It is something I have considered, so I might look into implementing it ;)
